tracer/test/test-applications/security/Samples.Security.AspNetCore5/blocking-template1.json

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE html>
```

Comment:
had some issues getting a file with anything other than `.json` extension being copied to the output directory.